### DISCUSSION TITLE
Error : keda-operator cannot list resource deployments

### DISCUSSION THREAD

**Discussion by shashanksdixit** [kedacore/keda#1588] [Q&A / Need Help] [answered]
We have deployed Keda usig helm, as given here on AKS (Azure Kubernetes Service)
https://keda.sh/docs/2.1/deploy/

We have created a sample example based on Azure service bus 
https://keda.sh/docs/2.1/scalers/azure-service-bus/

The scaledobject is essentially a receive application deployment, which needs to be scaled. 

The deployment is not scaling. When we checked the logs in keda-operator pod, we found one error.

`reflector.go:138] pkg/mod/k8s.io/client-go@v0.20.2/tools/cache/reflector.go:167: Failed to watch *v1.Deployment: failed to list *v1.Deployment: deployments.apps is forbidden: User "system:serviceaccount:keda:keda-operator" cannot list resource "deployments" in API group "apps" at the cluster scope`

The Keda operator should have the rights to list the deployments given at the time of Keda installation. This error should not have occurred. Can somebody help us point out what is missing?

Thanks in advance.

**Comment by tomkerkhove** [comment, marked as answer]
Did you, by any chance, use KEDA v2.1.0 Helm chart? Because we did a patch to support the RBAC issues in v2.1.1 - https://github.com/kedacore/charts/releases/tag/v2.1.1

**Reply by shashanksdixit** [reply]
Yes. I used the helm chart. Thank you for your reply. I uninstalled and installed Keda by yaml. And now the error is no more.

**Reply by tomkerkhove** [reply]
Great to hear and sorry for the trouble!
